roles/virthost-basics/tasks/main.yml

```diff
@@ -15,6 +15,8 @@
     - virt-install
     - genisoimage
     - nmap
+    - libvirt-python
```

Comment:
These are only needed when doing `with_ipv6` right?

Reply:
Somewhat. At least now these were necessary for the `virt_net` plays. So they might be otherwise useful in the future, at a fairly low cost of having the required packages on the virt host.